rtv/terminal.py

```diff
@@ -438,6 +439,20 @@ def open_editor(self, data=''):
             else:
                 _logger.info('File deleted: %s', filepath)
 
+    def open_urlview(self, data):
+        urlview = os.getenv('URLVIEWER') or os.getenv('RTV_URLVIEWER') \
```

Comment:
I made `$URLVIEWER` up because I couldn't find anything else :).  I'm happy enough to drop it for just `$RTV_URLVIEWER` to ensure we're not polluting the environment namespace